packages/lockfile-lint/__tests__/cli.test.js

```diff
@@ -59,7 +59,7 @@ describe('CLI tests', () => {
       expect(output).toContain(
         'detected invalid protocol for package: ms@^2.1.1\n    expected: https:\n    actual: http:\n'
       )
-      expect(output).toContain('error: command failed with exit code 1')
+      expect(output).toContain('Error: security issues detected!')
```

Comment:
I was aiming for a more informative error message. Open to feedback of course...

Reply:
Sure, that looks good and informative to me. Thanks!